Backend/index.js

```diff
@@ -390,6 +391,31 @@ app.post("/users/:id/follow", verifyToken, async (req, res) => {
       },
     });
 
+    // Fetch follower and following user details
+    const follower = await prisma.user.findUnique({
+      where: { id: parseInt(followerId) },
+    });
+
+    const followingUser = await prisma.user.findUnique({
+      where: { id: parseInt(followingId) },
+    });
+
+    // Create a notification for the user being followed
+    const notificationContent = `@${follower.username} just followed you.`;
+    const notification = await prisma.notification.create({
```

Comment:
is notification used anywhere? if not but prisma.notification.create is still needed, you don't need to assign this line to any variable